test/BountyRegistry.test.js

```diff
@@ -759,4 +686,632 @@ contract('BountyRegistry', function ([owner, user0, user1, user2, expert0, exper
       arbiter0Candidates[2].should.equal(user2);
     });
   });
+
+  describe('payouts', function() {
+    it('should refund bounty amount and bounty fee to ambassador if there are no assertions or votes', async function() {
+      let amount = ether(10);
+      let bid = ether(20);
+      let tx = await postBounty(this.token, this.bountyregistry, user0, amount, IPFS_README, 2, 10);
+      let guid = tx.logs[0].args.guid;
+
+      // wait for assertion period to pass
+      await advanceBlocks(10);
+
+      // wait for reveal period to pass
+      await advanceBlocks(ASSERTION_REVEAL_WINDOW);
+
+      await advanceBlocks(ARBITER_VOTE_WINDOW);
+
+      await settleBounty(this.bountyregistry, user0, guid);
+
+      let ambassadorBalance = await this.token.balanceOf(user0);
+      // init - bountyFee
+      ambassadorBalance.should.be.bignumber.equal(ether(STARTING_EXPERT_BALANCE));
```

Comment:
Can you rename/replace `STARTING_EXPERT_BALANCE` with `STARTING_AMBASSADOR_BALANCE`?

Reply:
I will create two vars